tv-shows-app/src/components/features/auth/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,99 @@
+'use client'
+import { apiPaths } from "@/app/data/api-paths";
+import { PasswordInput } from "@/components/shared/PasswordInput/PasswordInput";
+import { loginMutator } from "@/fetchers/mutators";
+import { ILogInFormInputs } from "@/typings/Auth.type";
+import { EmailIcon } from "@chakra-ui/icons";
+import { Button, Flex, FormControl, FormErrorMessage, Input, InputGroup, InputLeftElement, Text } from "@chakra-ui/react";
+import { useRouter } from "next/navigation";
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+import useSWRMutation from "swr/mutation";
+import NextLink from "next/link";
+import { useUser } from "@/hooks/useUser";
+
+
+export const LoginForm = () => {
+    const {register, handleSubmit, setError,
+        formState:{ 
+            isSubmitting,
+            errors
+        }
+    } = useForm<ILogInFormInputs>();
+    const [user, setUser] = useUser();
+    const router = useRouter();
+    const {trigger} = useSWRMutation(apiPaths.login, loginMutator, {
+        onSuccess: (data)=>{
+
+            const client = data.headers.get('client');
+            const accessToken = data.headers.get('access-token');
+            const uid = data.headers.get('uid');
+            const expiry = data.headers.get('expiry');
+
+            if(client && accessToken && uid && expiry){
+              setUser({
+                accessToken: accessToken,
+                client: client,
+                uid: uid,
+                expiry: expiry
+              });
+              router.replace('/shows');
+            }else{
+              setError("root",{type:"validate", message: "User data not complete."});
+            }
+        },
+        onError(err, key, config) {
+            setError("root", {type:"validate", message: err.message});
+        },
```

Comment:
Našlo se rješenje u kanalu na Slacku 💪 
```suggestion
        },
        throwOnError: false
```